sklearn/preprocessing/data.py

```diff
@@ -1878,13 +1878,14 @@ def _more_tags(self):
 
 
 class KernelCenterer(BaseEstimator, TransformerMixin):
-    """Center a kernel matrix
-
-    Let K(x, z) be a kernel defined by phi(x)^T phi(z), where phi is a
-    function mapping x to a Hilbert space. KernelCenterer centers (i.e.,
-    normalize to have zero mean) the data without explicitly computing phi(x).
-    It is equivalent to centering phi(x) with
-    sklearn.preprocessing.StandardScaler(with_std=False).
+    r"""Center a kernel matrix.
```

Comment:
In some previous issue, we agreed on not introducing latex docstring.
I am fine changing the `rst` file but if you should revert this docstring.

Reply:
Sorry, I didn't know about that. I know that numpydoc reference mentions that the summary should avoid LaTeX, but I thought that the rendition would be better.

I find that having some math without LaTeX is a bit confusing, but I will revert the docstring changes. Maybe the docstring could be modified to have less math and more words?

Reply:
> Sorry, I didn't know about that. I know that numpydoc reference mentions that the summary should avoid LaTeX, but I thought that the rendition would be better.

No problem. It is not easy to find that info :)

> I find that having some math without LaTeX is a bit confusing, but I will revert the docstring changes. Maybe the docstring could be modified to have less math and more words?

Maybe we don't need to mention the math equation at all and give the important message, i.e. what is a kernel trick. However, I would be interested to know what other devs are thinking about it.